gamemodes/terrortown/gamemode/server/sv_main.lua

```diff
@@ -253,8 +253,12 @@ util.AddNetworkString("TTT2RoleGlobalVoice")
 util.AddNetworkString("TTT2MuteTeam")
 util.AddNetworkString("TTT2UpdateHoldAimConvar")
 
-fileloader.LoadFolder("terrortown/gamemode/client/cl_help/", false, CLIENT_FILE)
+-- provide menu files by loading them from here:
+fileloader.LoadFolder("terrortown/menus/score/", false, CLIENT_FILE)
+fileloader.LoadFolder("terrortown/menus/gamemode/", false, CLIENT_FILE)
+fileloader.LoadFolder("terrortown/menus/gamemode/", true, CLIENT_FILE)
```

Comment:
Shouldn't deepsearch include non deepsearched files?

Reply:
It doesn't no. I thought about changing it, but this would then also search for two layers in the language folder. And this is unnessecary.

Reply:
Hmm okay, that's definitely unusual :D